src/com/epam/javacourse/homework2/Fibonacci.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.javacourse.homework2;
+
+public class Fibonacci {
+    public void getFibonacciRow(int n) {
+        int first = 0;
+        int second = 1;
+        System.out.println("Fibonacci Series:");
```

Comment:
Let's replace this line with `System.out.print("Fibonacci Series: ");` for a prettier output.

Reply:
it's not obligatory to change
just to mention that it could be a one-line output like in other tasks

Reply:
Discussed.